examples/db-templates/mongodb-ephemeral-template.json

```diff
@@ -214,5 +214,6 @@
   ],
   "labels": {
     "template": "mongodb-ephemeral-template"
-  }
+  },
+  "message": "You can connect to the database using MongoDB connection URL mongodb://${MONGODB_USER}:${MONGODB_PASSWORD}@${DATABASE_SERVICE_NAME}/${MONGODB_DATABASE}"
```

Comment:
are we sure this url format is generic to all frameworks?  eg on eap i'd expect a jdbc:// url.